packages/squiggle-lang/src/fr/tag.ts

```diff
@@ -237,6 +239,38 @@ example2 = {|x| x + 1}`,
       }),
     ],
   }),
+  maker.make({
+    name: "specification",
+    description: `Adds a specification to a value. This is useful for documenting how a value was calculated, or what it represents.`,
+    displaySection: "Tags",
+    definitions: [
+      makeDefinition(
+        [frWithTags(frAny({ genericName: "A" })), frSpecification],
+        frWithTags(frAny({ genericName: "A" })),
+        ([{ value, tags }, spec], reducer) => {
+          const showAs = spec.showAs && reducer.call(spec.showAs, [value]);
+          return {
+            value,
+            tags: tags.merge({
+              specification: vSpecification(spec),
+              name: vString(spec.title),
```

Comment:
We often don't have much horizontal space - the value can get quite small, especially when nested.

It seems like this is an issue when:
1. The user doesn't set a different name, with a tag.
2. The user would prefer to see the variable name, instead of the spec name.

This seems less likely to me than them preferring to see the spec name.